util/formula_templater/formula.go

```diff
@@ -68,18 +68,24 @@ const formulaTemplate = `class {{ .Name }} < Formula
   version "{{ .Version }}"
 
   {{- if .Architectures.DarwinAmd64 }}
+  {{- if .Architectures.DarwinArm64 }}
```

Comment:
Nitpick: Have you considered merging the two conditions into one?

```suggestion
  {{- if and .Architectures.DarwinAmd64 .Architectures.DarwinArm64 }}
```

Reply:
Ah thanks this is more clear. I was trying to figure out how to do this on https://golang.org/pkg/text/template/ and couldn't find the answer